config/v1/types_infrastructure.go

```diff
@@ -101,6 +101,15 @@ type InfrastructureStatus struct {
 	// +kubebuilder:default=HighlyAvailable
 	// +kubebuilder:validation:Enum=HighlyAvailable;SingleReplica
 	InfrastructureTopology TopologyMode `json:"infrastructureTopology"`
+
+	// cpuPartitioning expresses that CPU partitioning is a currently enabled feature in the cluster.
+	// CPU Partitioning means that this cluster can support partitioning workloads to specific CPU Sets.
+	// Valid values are None and AllNodes. When omitted, the default value is None.
```

Comment:
What's the difference in behaviour between `None` and `AllNodes`? Can you explain briefly in the Godoc?

Reply:
Sure thing, please give it another look if that adds enough context around this field.